x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/_layer_panel.scss

```diff
@@ -27,6 +27,14 @@
   overflow: hidden;
 }
 
+.lnsLayerPanel__dimension-isHidden {
+  opacity: 0;
+}
```

Comment:
It looks like this is no longer working because the specificity of the `.lnsDragDrop.lnsDragDrop-isNotDroppable` is overriding it. It would be better if this was part of the DnD classes so it can also be reused in other places.
<img width="522" alt="Screen Shot 2020-08-31 at 12 41 10 PM" src="https://user-images.githubusercontent.com/549577/91744376-683a8780-eb87-11ea-98b1-854130dece10.png">

Reply:
Fixed